commerce-account-web/src/main/java/com/liferay/commerce/account/web/internal/frontend/CommerceAccountAddressClayTable.java

```diff
@@ -109,11 +108,18 @@
 		sb.append(StringPool.SEMICOLON);
 
 		ClayTableAction deleteClayTableAction = new ClayTableAction(
-			sb.toString(), StringPool.BLANK,
-			LanguageUtil.get(httpServletRequest, "delete"), false, false);
+			"javascript:deleteCommerceAddress" + sb.toString(),
```

Comment:
Rather than manually appending these strings, it's probably better to keep the sb as length=7, and just resetting the first element of the sb before using it for `editClayTableAction`, like `sb.setStringAt("javascript:editCommerceAddress", 0);`